CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+
+## v1.2.1
+
+In the fixing of three bugs, three minor features have been added along the way.
```

Comment:
do we now start to have taglines per release? ;-)

Reply:
Just felt the need to justify there being some features in a patch release. You think I should remove it?

Reply:
It's fine :-)